handbook/business-operations/README.md

```diff
@@ -579,9 +584,12 @@ The steps for doing this are highlighted in this loom, TODO.
 6. Run contractor payroll
 
 #### Commission payroll
+
+> TODO: bit more process here.  Maybe revops is DRI of commission calculator, creates "2023-03 commission payroll", transfered to Nathan when it's time to run?  SLA == payroll run by the 7th, with commission sheet 100% accurate.
```

Comment:
@hollidayn think you could show @hughestaylor how to set up an issue template for commission payroll?

Reply:
@hollidayn wait nvm it’s already part of monthly close.

So I’m wrong here!  No new issue template needed.  Does that just mean @hughestaylor should be the one to kick off monthly close?  Then he hands it to Nate?

Reply:
Worth discussing at stand up tomorrow!

@hughestaylor also had some ideas on the commission calculator I think are worth hearing out - could simplify keeping track of everything for us and make it easier to run commissions!